src/payment/strategies/bolt/bolt-payment-initialize-options.ts

```diff
@@ -1,13 +1,56 @@
+/**
+ * A set of options that are required to initialize the Bolt payment method with:
+ *
+ * 1) Bolt Full Checkout:
+ *
+ * If the customer chooses to pay with Bolt, he will be asked to
+ * enter his payment details via Bolt Full Checkout.
+ *
+ * ```js
+ * service.initializePayment({
+ *     methodId: 'bolt',
+ * });
+ * ```
+ *
+ * 2) Bolt Client:
+ *
+ * If the customer chooses to pay with Bolt, he will be asked to
+ * enter his payment details via Bolt modal.
+ *
+ * ```js
+ * service.initializePayment({
+ *     methodId: 'bolt',
+ *     bolt: {
+ *         useBigCommerceCheckout: true,
```

Comment:
Just one question, is this always `true`. The `useBigCommerceCheckout: true` implies that the customer is going to use Bigcommerce checkout but from description feels like the customer will be using Bolt checkout.

Reply:
Yes, for this case it always true. 

Bolt payment provider has 3 different flows:
1) Full Checkout - customer pass checkout on the Bolt side;
2) Fraud protection only - customer pass the BC checkout but has bolt payment modal open when he select bolt in payments section;
3) Embedded one Click - is a flow with embedded fields

Reply:
Updated the description to make it easier to understand